app/controllers/tea_times_controller.rb

```diff
@@ -32,6 +33,7 @@ def new
 
   # GET /tea_times/1/edit
   def edit
+    use_new_styles
```

Comment:
remove this and also remove `:edit` from list of exceptions in `before_action` at the top of this file.